scripts/recovery_sync

```diff
@@ -9,6 +9,12 @@
 
 set -euxo pipefail
 
+function cleanup() {
+	rm -rf "$workdir"
+	rm "${target}.tmp"
+
+trap cleanup EXIT
```

Comment:
Since the cleanup function depends on the `workdir` and `target` variables, it might be best to move this call to some point after we've initialized those two variables.. e.g. otherwise, when the wrong number of parameters are passed, I think the cleanup function may result in more errors (due to these variables being undefined at that point).